cumulusci/tasks/release_notes/tests/test_parser.py

```diff
@@ -549,3 +550,26 @@ def _create_expected_render(self, issue_number, issue_title, link_pr):
         if link_pr:
             render += " [[PR{}]({})]".format(self.pr_number, self.pr_url)
         return render
+
+
+class TestGithubLinesParser(unittest.TestCase):
```

Comment:
```suggestion
class TestInstallLinkParser:
```

Reply:
Please make sure your editor is set up to run flake8 for Python files. It should have warned you that you were re-defining an existing class.